designs/design-template.stanza

```diff
@@ -0,0 +1,49 @@
+#use-added-syntax(jitx)
+defpackage design-template :
+  import core
+  import jitx
+  import jitx/commands
+
+; =========================
+; Module to run as a design
+; ==========================
+val board-shape = RoundedRectangle(30.0, 18.5, 0.25)
+pcb-module my-design :
+
+  inst version-label  : ocdb/board-text/version-silkscreen("REV 0.0.0")
+  place(version-label) at loc(0.0, height(board-shape) / 2.0 - 1.0) on Bottom
+  
+; ========================================
+; Configure the design and view the result
+; ========================================
+defn run-design (circuit:Instantiable, outline:Shape) :
+  set-design("CAD")
```

Comment:
Some comments here might be useful : 

```
  set-design("CAD") ; name our design, which creates an output directory of the same name.

  set-main-module(circuit) ; declare the "main" module, analogous to a root schematic sheet (phrasing?)
  set-rules(ocdb/rules/jjcpcb-rules) ; etc 
```

Reply:
I'm going to change `set-design` to `set-design-directory` instead of adding a comment.